app/views/layouts/listnav/_ems_middleware.html.haml

```diff
@@ -5,8 +5,11 @@
 
     = miq_accordion_panel(_("Properties"), false, "ems_middleware_prop") do
       %ul.nav.nav-pills.nav-stacked
-        %li
-          = link_to(_('Summary'), {:action => 'show', :id => @record, :display => 'main'}, :title => _("Show Summary"))
+        = li_link(:if => true,
```

Comment:
The thing here is, I always want this to be enabled without having any particular condition if I don't put true here this link is gonna be disabled per default (I see this here: https://github.com/ManageIQ/manageiq/blob/3921e87915b5a69937b9d4a70bb24ab71b97c165/app/views/layouts/listnav/_ems_network.html.haml#L8), they used this helper in the same way.